homeassistant/components/reolink/binary_sensor.py

```diff
@@ -130,7 +129,12 @@ def __init__(
         self.entity_description = entity_description
 
         if self._host.api.model in DUAL_LENS_DUAL_MOTION_MODELS:
-            self._attr_name = f"{entity_description.name} lens {self._channel}"
+            key = (
+                entity_description.translation_key
+                if entity_description.translation_key is not None
+                else entity_description.key
+            )
```

Comment:
- Please don't make multi-line ternaries
- This cannot be correct, for example, this will set the `motion` as translation key for the motion sensor, while in that case, we should not set a translation key and fallback to the translations provided by the device class.

Reply:
I fixed the multi-line ternarie, indeed that was not nice.

For the motion sensor it will set the translation key to `motion_lens_0` and `motion_lens_1` not `motion`, see the next line.

Note that this piece of (anoying) code is only applicaple to the DUO v1 model which is diffrent to all other 32 models in that it has two lenses in a single camera with seperate motion/AI detection for both lenses.
Rest of the entities are only 1 which apply to both lenses like daynight-mode, floodlight, detection sensitivity etc.

Reply:
Aah check, thanks 👍